test/jit/test_string_formatting.py

```diff
@@ -0,0 +1,156 @@
+import os
+import sys
+
+import torch
+from typing import List
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+class TestStringFormatting(JitTestCase):
+
+    def test_modulo_operator(self):
+        @torch.jit.script
+        def fn(dividend: int, divisor: int) -> int:
+            return dividend % divisor
+        self.assertEqual(1, fn(5, 2))
+
+    def test_string_interpolation_with_string_placeholder_and_string_variable(self):
+        @torch.jit.script
+        def fn(arg1: str):
+            return "%s in template" % arg1
+        self.assertEqual("foo in template", fn("foo"))
```

Comment:
Is TorchScripted percentFormat generating exactly same output as native python? It would be better if we can use `self.checkScript` in these tests to ensure eager and TS behavior match.